model/security/get_tenant/operations.smithy

```diff
@@ -0,0 +1,39 @@
+//  SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+use opensearch.openapi#vendorExtensions
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/2.7/security/access-control/api/#get-tenant"
+)
+
+@vendorExtensions(
+    "x-operation-group": "security.get_tenants",
+    "x-version-added": "1.0",
+)
+@readonly
+@suppress(["HttpUriConflict"])
+@http(method: "GET", uri: "/_plugins/_security/api/tenants/")
+@documentation("List of all tenants.")
+operation GetTenants {
+    input: GetTenants_Input,
+    output: GetTenants_Output
+}
+
+@vendorExtensions(
+    "x-operation-group": "security.get_tenant",
```

Comment:
this [model/security/get_tenant/](https://github.com/opensearch-project/opensearch-api-specification/pull/113/files#diff-774b3f091e8939f7ab874e3c81bf18409f1d6c13bb002fc529937a68f820e810) folder actually hosts 2 different operation groups: `security.get_tenants` and `security.get_tenant`. There should only be one group per folder. And don't forget to add  the correct "API Reference" for each.

Reply:
Note that these 2 groups might share some common structures in the bodies. You only need to define it in one. Say define `Tenant` in `security/get_tenant` and you can reuse it in `security/get_tenants`